### DISCUSSION TITLE
Add an option to make it like Windows Notifications Area

### DISCUSSION THREAD

**Discussion by Lainerai** [martinpl/tray-icons-reloaded#45] [Ideas]
Like this..
![image](https://user-images.githubusercontent.com/33560586/122371784-b6900a00-cf92-11eb-9bef-cb1095ff8434.png)
is it possible..?

**Comment by martinpl** [comment]
Not 1:1 same but can do that by setting icons limit to 1. I should highlight it in readme.
![image](https://user-images.githubusercontent.com/8329810/122644358-17dfe680-d115-11eb-960d-4e2128851269.png)

**Reply by Lainerai** [reply]
Oh sorry I haven't played around with the icons limit so I didnt know, no need to make it 1:1 actually, it's great. Now my panel looks more clean, thanks.

**Reply by Lainerai** [reply]
btw sir, I just cant help my curiousity so, what is this?
![image](https://user-images.githubusercontent.com/33560586/122645732-62318980-d14e-11eb-8cb9-9528db24d920.png)

**Reply by martinpl** [reply]
It is a [workspace indicator](https://extensions.gnome.org/extension/3952/workspace-indicator/) ;)

**Reply by Lainerai** [reply]
aaah, ok thanks sir.
